tools/cli/flags.go

```diff
@@ -162,6 +162,8 @@ const (
 	FlagClusterMembershipRole                 = "role"
 	FlagIsGlobalNamespace                     = "global_namespace"
 	FlagIsGlobalNamespaceWithAlias            = FlagIsGlobalNamespace + ", gd"
+	FlagPromoteNamespace                      = "promote_namespace"
+	FlagPromoteNamespaceWithAlias             = FlagPromoteNamespace + ", pn"
```

Comment:
why not make `is global namespace` configurable

Reply:
It is due to the proto field does not support *bool, so if we use is_global, we would force user to specify that for any namespace update, and we have to deal with change from true to false which we don't support.